Cakes-Co/index.html

```diff
@@ -33,10 +33,9 @@
       <section class="p-sec">
         <h1>Welcome</h1>
         <p>
-          Lorem ipsum dolor sit amet consectetur, adipisicing elit. Quae
-          voluptatem accusantium, necessitatibus cum accusamus, id cumque harum
-          voluptates ipsum nisi vero illo dolore! Doloribus, quae hic. Facilis
-          voluptate nobis eum.
+          we believe that every celebration deserves a cake that is as unique and special as the occasion itself.
```

Comment:
It looks like there are files in this PR that don't belong to this project. Can you open a PR with just the changes for Form Controls, please? Come to class to get help with this if you are stuck.